domainobjs/ts/publicKey.ts

```diff
@@ -19,11 +19,18 @@ export class PubKey {
 
   /**
    * Create a new instance of a public key
+   * @dev You might want to allow an invalid raw key,
```

Comment:
I think I don’t quite understand this part. Why decrypting invalid message should use invalid raw key?

Reply:
Basically imagine you send an invalid ciphertext (the encrypted message) and a key which wasn't used to encrypt it. when decrypting the message forcefully, the data will be invalid because the key is not able to decrypt it, but we still choose to get the decryption result. And because we need to forcefully decrypt messages here https://github.com/privacy-scaling-explorations/maci/blob/dev/core/ts/Poll.ts#L554 the `Command.Decrypt` operation would fail when creating the public key https://github.com/privacy-scaling-explorations/maci/blob/dev/domainobjs/ts/commands/PCommand.ts#L210